Kiosk/App/BidderDetailsRetrieval.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+extension UIViewController {
```

Comment:
Why are these being added to UIViewController and UIAlertController? And is this extension by default scoped to application code only? (i.e. no collisions can occur?)